libcob/coblocal.h

```diff
@@ -570,6 +570,9 @@ cob_max_int (const int x, const int y)
 	return y;
 }
 
+COB_HIDDEN int		common_cmps	(const unsigned char *, const unsigned char *,
```

Comment:
so far all `COB_HIDDEN` have a `cob_` prefix. please refactor this to be identical